### PR TITLE
 #19455:changed hard coded values to constants

### PR DESCRIPTION
**Pull request by knoxy5467**

### CODE REVIEW

**Comment by knoxy5467** [comment]
I've mad errors, Will FIx

**Comment by spacek531** [comment]
You don't need to close the PR to fix it. You can mark it as draft to indicate that you are still working on it, then switch it to ready for review when you feel it's done.

**Comment by knoxy5467** [comment]
Thank you for the information, I'm still learning about contributing to open source projects.

**Comment by ZehMatt** [comment]
Seems reasonable, as a first time contributor you might want to add yourself to contributors.md found in the root folder.

File: src/openrct2-ui/windows/EditorObjectiveOptions.cpp [hunk 1]
```diff
@@ -37,6 +37,16 @@ static constexpr const StringId WINDOW_TITLE = STR_OBJECTIVE_SELECTION;
...
 
+const money64 ObjectiveMaxCurrency = 2000000.00_GBP;
+const int ObjectiveMaxLength = 5000;
+const money32 ObjectiveMinCurrency = 1000.00_GBP;
+const int ObjectiveMinLength = 1000;
+const int ObjectiveMaxExcitement = FIXED_2DP(9, 90);
+const int ObjectiveMinExcitement = FIXED_2DP(4, 00);
+const short ObjectiveMinGuests = 250;
+const short ObjectiveMaxYear = 25;
+const short ObjectiveMinYear = 1;
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/EditorObjectiveOptions.cpp:48]: Can you try making these `constexpr` as much as possible?

**Pull request closed without merging**